lib/net/pool.js

```diff
@@ -650,7 +661,10 @@ class Pool extends EventEmitter {
       return;
 
     this.syncing = true;
-    this.resync(false);
+    if (this.options.neutrino) {
+      this.startHeadersSync();
+    } else
+      this.resync(false);
```

Comment:
I don't think you need this switch here as long as we request headers (not blocks) in `sendLocator` (see https://github.com/pinheadmz/bcoin/commit/b7cadf44e1460e823f765f370cba7216f52c3271)

Reply:
resolved